Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM wordpress:latest
 
-RUN pecl install xdebug \
-    && docker-php-ext-enable xdebug
+# RUN pecl install xdebug \
+#     && docker-php-ext-enable xdebug

```

Comment:
do we want this commented out? Maybe write a comment on top saying to uncomment if xdebug is needed or something like that? 🙃

Reply:
It's messing up the docker compose on osx somehow.. Maybe I should revert this for now.